.env

```diff
@@ -25,3 +25,6 @@ SENDWYRE_ACCOUNT_ID=AC_ERP9DMNTAMB
 SENDWYRE_ENVIRONMENT=test
 MOONPAY_KEY=pk_test_6IqwtCJDawLWNcmzaTeBfE4mBF0GwE4
 WIDGET_SIGNATURE=gFg8YE4qZpvya75B
+BITCOIN_NETWORK=testnet
+# BITCOIN_INSIGHT_URL=http://3.8.237.170:3001/insight-api
+BITCOIN_INSIGHT_URL=https://test-insight.bitpay.com/api
```

Comment:
How does are different? Why other .envs contains only the one with IP address?

Reply:
The IP address is our own node, but it doesn't always work, so we use bitpay's node for now.

Reply:
Can we add a comment here?

Reply:
Okay, added a comment.